client/my-sites/upgrades/checkout/thank-you.jsx

```diff
@@ -346,14 +349,16 @@ JetpackPremiumPlanDetails = React.createClass( {
 					title={ this.translate( 'Akismet' ) }
 					description={ this.translate( 'Say goodbye to comment spam' ) }
 					buttonText={ this.translate( 'Start using Akismet' ) }
-					onButtonClick={ goToExternalPage( 'https://en.support.wordpress.com/setting-up-premium-services/' ) } />
+					href="https://en.support.wordpress.com/setting-up-premium-services/"
```

Comment:
I know this is not the purpose of this pull request but we could use this opportunity to remove the `en` locale prefix from these support links.